index.html

```diff
@@ -20,14 +20,20 @@
 <body>
     <header class="nav-container">
         <div class="navbar-list">
-            <a href="#">
-                <h3 class="name1">AB</h3>
-            </a>
+            <h3 class="name1"><a href="#">AB</a></h3>
             <i class="hamburger-menu fa fa-bars"></i>
+            <ul class="hamburger-list">
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li>
+                    <a href="#"><i class="fa fa-envelope-o"></i> </a>
+                </li>
+            </ul>
         </div>
     </header>
     <section class="header-container-top">
```

Comment:
- [ ] you are missing the work indicator and social icons in your desktop version. kindly fix that
your version
![Screenshot (218)_LI](https://user-images.githubusercontent.com/65298048/177863764-3ae319f8-d9f9-4d18-b596-482c9d86fbb2.jpg)
figma version
![Screenshot (219)_LI](https://user-images.githubusercontent.com/65298048/177863887-c16d0c45-3a99-469e-9589-178af3b7bce5.jpg)